archivers/unshield/Portfile

```diff
@@ -12,25 +13,23 @@ long_description \
 	Extract files from InstallShield-style cabinet files (usually named \
 	data1.cab, data1.hdr, data2.cab, etc), versions 5 and later
 
+homepage			https://github.com/twogood/unshield
+master_sites		https://github.com/twogood/unshield/archive/refs/tags/
+distfiles           ${version}${extract.suffix}
 
-homepage			http://synce.sourceforge.net/synce/unshield.php
-master_sites		sourceforge:synce
+depends_lib			port:libiconv \
+                    port:zlib
 
-depends_lib			port:zlib
+checksums			md5     ba0759866923a62be5ca799232c2b480 \
+					sha256  aa8c978dc0eb1158d266eaddcd1852d6d71620ddfc82807fe4bf2e19022b7bab \
+					rmd160  0f26f1c0d3637762fa06a3c2d60253046f0ed085 \
+                    size    58822
```

Comment:
you might need to update the checksums (`sudo port clean --all unshield && sudo port -dv checksum unshield`); the `md5` field can be dropped, we don't use that anymore.

Reply:
I've removed the md5 checksum. I've validated the other checksums. They didn't change after I made the suggested changes to the portfile.